packages/datadog-instrumentations/src/cookie-parser.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const { AbortController } = require('node-abort-controller') // AbortController is not available in node <15
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook } = require('./helpers/instrument')
+
+const cookieParserReadCh = channel('datadog:cookie-parser:read:finish')
+
+function publishRequestCookieAndNext (req, res, next) {
+  return function cookieParserWrapper () {
+    if (cookieParserReadCh.hasSubscribers && req) {
+      const abortController = new AbortController()
+
+      cookieParserReadCh.publish({ req, res, abortController, cookies: req.cookies || req.signedCookies })
```

Comment:
I think that you can't choose between cookies and signedCookies, you should send both, maybe in a single merged object or in two objects

It looks like req.cookies is always present, but signedCookies not.
https://github.com/expressjs/cookie-parser/blob/master/index.js#L60-L69

Reply:
TODO: dig this more and do some testing

Reply:
Done, I've digged into it and realized I misunderstood the cookie-parser code, I've added an `Object.assign()` to merge the two objects into one